### PR TITLE
Pressurized Slime: Buff to oxygen and SILICON components

### PR DESCRIPTION
**Pull request by carshalash**
Water is fucking dumb to slip on now, even if some retard manages to slip on the wet floors, it's only a one tile difference and within range to pick up their guns easily. Lets fucking change that, pressurized blob is now the clowns best friend and will lube tiles.

:cl: Carshalash
   rscadd: Replaces Pressurized Slime to spray lube instead of water
   /:cl:

### CODE REVIEW

**Comment by subject217** [comment]
Wow, another silicon buff from Carshalash :wink:

**Comment by Barhandar** [comment]
Pressurized Slime will be the best blob chem, even better than cryo poison.

**Review by 81Denton — CHANGES_REQUESTED** [review]
Change all the description lines too por favor

**Comment by carshalash** [comment]
> Change all the description lines too por favor

Fix the tubes to the ai sat on meta REEEEEEEEEEEEEEEEEEEEEEE

**Comment by carshalash** [comment]
"	description = "will do low brute, oxygen, and stamina damage, and wet tiles under targets."

You mean that right?

**Comment by 81Denton** [comment]
>Fix the tubes to the ai sat on meta REEEEEEEEEEEEEEEEEEEEEEE

Odd, I'll have another look at them.

>You mean that right? I thought it still applied pretty well so I left it.

Can you replace the "wets" part in those four desc lines with "lubes"? Lube doesn't freeze into ice and slips harder, I'd rather the exact name to be sure.

**Comment by carshalash** [comment]
Now give me back the transit tubes you can walk over, you removed them all reeeeee.

**Comment by ExcessiveUseOfCobblestone** [comment]
`Buff to oxygen and SILICON components` what does this mean?

**Comment by Barhandar** [comment]
Spacelube is made with oxygen, silicon and water reagents, also silicons are slip-proof.

**Comment by ExcessiveUseOfCobblestone** [comment]
cringe

**Comment by zxaber** [comment]
People will just disable gravity, like they did before the slip rework.

**Comment by carshalash** [comment]
> People will just disable gravity, like they did before the slip rework.

Better than people just not needing to worry at all about it.

**Comment by zxaber** [comment]
The result is the only positive effect of this chem gets bypassed by engineer borgs, moths, anyone with mag boots, and anyone with a ranged weapon. That would be fine if blob wasn't a random roll of the dice for a decent chem. As long as changing chems is completly random, it's not great to have one so drastically useless (or at least, so situational).

**Comment by Barhandar** [comment]
Magboots slip on lube, no? Also, still better than water (which IS bypassed by everything you've listed, but doesn't even slip satisfactorily).

**Comment by zxaber** [comment]
While gravity is out? I have no idea. My understanding was that gravity disables slips entirely, but I suppose mag boots might change that.

Anyway, you're right, the current version is even worse, but I don't see this as a solid fix. Honestly, I think oranges was right, we ought to remove this chem entirely. If we could balance around the blob getting to choose the chem, it would be a nice situational choice, but unless you're specifically running from a chem that got hardcountered, chances are what you had before gave a better bonus than forcing the crew to fight without gravity.

**Comment by carshalash** [comment]
Lube will leave the blob pretty powerful when it comes to  closed maint areas , it won't be good in spaced places but I see this being really decent. It's not gonna be emp levels of good but it'll still be pretty fun.

**Comment by YakumoChen** [comment]
Raise the damage too to make up for the fact it doesn't stun now

**Comment by carshalash** [comment]
Raise its stamina/oxygen damage or brute damage?

File: code/modules/reagents/chemistry/reagents/blob_reagents.dm [hunk 1]
```diff
@@ -470,7 +470,7 @@
 	reac_volume = ..()
 	var/turf/open/T = get_turf(M)
 	if(istype(T) && prob(reac_volume))
-		T.MakeSlippery(TURF_WET_WATER, min_wet_time = 10 SECONDS, wet_time_to_add = 5 SECONDS)
+		T.MakeSlippery(TURF_WET_LUBE, min_wet_time = 10 SECONDS, wet_time_to_add = 5 SECONDS)
 		M.adjust_fire_stacks(-(reac_volume / 10))
 		M.ExtinguishMob()
 	M.apply_damage(0.4*reac_volume, BRUTE)
```
Comment (YakumoChen) [inline code/modules/reagents/chemistry/reagents/blob_reagents.dm]: I'd change it to 0.5 maybe? Most of the blob's damage is temporary so if someone pulls out the victim before they crit they're usually fine (which almost always happens unless they're attacking from a lone angle)

**Comment by carshalash** [comment]
hm... how do the number values here even work in comparison to their wiki numbers?

**Comment by ExcessiveUseOfCobblestone** [comment]
Changelog needed

**Comment by carshalash** [comment]
> Changelog needed
> 
> Also if you are editing anything beyond making the slip go from water-like to lube-like please update OP (preferably in the templated format).

Still contemplating whether I should touch damage or leave it be to see how it works with the lube changes.

**Comment by carshalash** [comment]
Yeah I think i'll leave damage as is at the moment. Maybe tweak it in a future PR involving other blob changes.

**Pull request merged**